components/widgets/climate/emissions-deforestation/selectors.js

```diff
@@ -39,76 +39,77 @@ export const parseData = createSelector(
     return (
       zeroFilledData &&
       zeroFilledData
-        .filter(d => d.year >= startYear && d.year <= endYear)
-        .map(d => ({
+        .filter((d) => d.year >= startYear && d.year <= endYear)
+        .map((d) => ({
           ...d,
-          co2LossByYear: d.emissions,
-          biomassLoss: d.biomassLoss
+          emissions: d[emissionType],
         }))
     );
   }
 );
 
-export const parseConfig = createSelector(
-  [getSettings, getColors],
-  (settings, colors) => {
-    const { unit } = settings;
-    const { loss } = colors;
-    return {
-      height: 250,
-      xKey: 'year',
-      yKeys: {
-        bars: {
-          [unit]: {
-            fill: loss.main,
-            background: false
-          }
-        }
+export const parseConfig = createSelector([getColors], (colors) => {
+  const { loss } = colors;
+  return {
+    height: 250,
+    xKey: 'year',
+    yKeys: {
+      bars: {
+        emissions: {
+          fill: loss.main,
+          background: false,
+        },
       },
-      xAxis: {
-        tickFormatter: yearTicksFormatter
+    },
+    xAxis: {
+      tickFormatter: yearTicksFormatter,
+    },
+    tooltip: [
+      {
+        key: 'year',
       },
-      tooltip: [
-        {
-          key: 'year'
-        },
-        {
-          key: [unit],
-          unit: 't',
-          unitFormat: value => format('.3s')(value)
-        }
-      ],
-      unit: 't',
-      unitFormat: value => format('.2s')(value)
-    };
-  }
-);
+      {
+        key: 'emissions',
+        unit: 't CO2e',
+        unitFormat: (value) => format('.3s')(value),
+        color: loss.main,
```

Comment:
@tomasmoose this isn't working for some reason - any idea?

Reply:
resolved, `label` field required